docs/dom-testing-library/api-events.md

```diff
@@ -93,3 +93,31 @@ const myEvent = createEvent.click(node, { button: 2 })
 fireEvent(node, myEvent)
 // myEvent.timeStamp can be accessed just like any other properties from myEvent
 ```
+
+## Using Jest Function Mocks
+[Jest's Mock functions](https://jestjs.io/docs/en/mock-functions) can be used to test that a callback passed to the function was called, or what it was called when the event that **should** trigger the callback function does trigger the bound callback.
+
+```jsx
+const Button = ({onClick,children}) => <button onClick={onClick}>{children}</button>
+describe('Button component', () => {
```

Comment:
I prefer to [avoid nesting](https://www.briefs.fm/3-minutes-with-kent/27). We already know that this is about the button component because we're in the file that's testing the button component.